### PR TITLE
packetfence-perl packages

### PR DESCRIPTION
**Pull request by nqb**
# Description
Code to build packetfence-perl packages (EL and Debian)

# Delete branch after merge
NO

### CODE REVIEW

File: addons/cpan_perl/install_cpan.sh [hunk 1]
```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+# ===== USAGE =====
+# Usage: $ create_csv-dep_file.sh dependencies.csv
```
Comment (nqb) [inline addons/cpan_perl/install_cpan.sh]: Wrong usage

**Comment by nqb** [comment]
Packaging adjusted. Rebase done.

I built two new packages which are now in 11.0 repositories:
* `packetfence-perl-1.1.0-1.el8.x86_64.rpm`
* `packetfence-perl_1.1.0_all.deb`

Revisions in packaging will change when we will build packages using CI.

@JeGoi, if everything is ok for you, I think we should merge this one before the release.

**Comment by nqb** [comment]
I run a pipeline: https://gitlab.com/inverse-inc/packetfence/-/pipelines/353778896
I will check both packages have been installed and configurator test passed.

File: addons/packetfence-perl/README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+./install.sh dependencies.csv
+```
+
+Logs are available in `/root/install_perl` directory.
+
+## How to build RPM package
+
+``` shell
+./make_tar_from_source.sh
+QA_RPATHS=$(( 0x0001 )) rpmbuild -ba ./rhel8/SPECS/cpan_perl_module.spec
```
Comment (JeGoi) [inline addons/packetfence-perl/README.md:106]:
```suggestion
rpmbuild -ba ./rhel8/SPECS/cpan_perl_module.spec
# On Docker, you should have to specified the QA_RPATHS
QA_RPATHS=$(( 0x0001 )) rpmbuild -ba ./rhel8/SPECS/cpan_perl_module.spec
```

**Review by JeGoi — COMMENTED** [review]
Except the docker part.
It looks fine for me.

**Comment by nqb** [comment]
> Except the docker part.

Too late to add automation related to Docker. But I built new packages in same environment as we will use in CI.

I prefer we merge as-is and I will create a new PR to support build in CI.

**Pull request merged**